### ISSUE TITLE
Stuck on syncing. upgrade to 1.1.2 doesn't help

### ISSUE THREAD

**Issue by ardabro** [Chia-Network/chia-blockchain#2919]
This issue is permanent for the last couple of hours. I have a connection, tip block number increases, but the block I'm synced with doesn't progress at all:
```
Current Blockchain Status: Full Node syncing to block 194446 
Currently synced to block: 193536
Current Blockchain Status: Not Synced. Peak height: 193536
      Time: Tue Apr 27 2021 11:42:21 CEST                  Height:     193536
```

**Comment by lvcivs** [comment]
Do you see any connecttions to peers?

Have you configured UPnP in your router or configured port forwarding?

Have you tried stopping chia, deleting `~/.chia/mainnet/db/peer_table_node.sqlite`, and restarting?

**Comment by ardabro** [comment]
I'm connected with many peers. I was farming for the last couple of weeks without issues. 
`chia show --connections ` shows 52 connections with peer nodes.
Restart doesn't help. I have no crashed/zombie chia processes

The current status (still on the same block 193536):


Current Blockchain Status: Full Node syncing to block 194066
Currently synced to block: 193536
Current Blockchain Status: Not Synced. Peak height: 193536
      Time: Tue Apr 27 2021 11:42:21 CEST                  Height:     193536

Estimated network space: 907.359 PiB
Current difficulty: 54
Current VDF sub_slot_iters: 110624768
Total iterations since the start of the blockchain: 635507339856

  Height: |   Hash:
   193536 | 85aa575df9e1569a8930e6162cf08514c00efbcd8a4cffc259b4f215c6ed570c
   193535 | 92254bd4886c86ec604fa0f0241816704fa4a1c64a11f12383db854ccaf9321c
   193534 | fb66324b32670627a154c80f27181af86e47c1ba1a36075cb6a6caab2e60584e
   193533 | 3a03e75feaf6c2d73deb187ab5571d91a4ad1e5819072b77b8554f629abc9a99
   193532 | 2c657226da5db848e986b6331dfff2c205a2ee7e8c80fe335dc4f8db92127615
   193531 | 58a27b69bfc98c7122ef6aa277450cc3b0d5c9deb1d70816d29b45573ef4130c
   193530 | 005bb7dc06f6a700862e720eccb683c7e1197cc3bed913b3e4a6ca93879ae195
   193529 | 84aca63df61980f5ba6fe5227deb770dcd609542c5992f258fa5b9d7b070c961
   193528 | 5b9c28735ceebb496987f3686bede7a58023a2c3566deda64d1135deebad6070
   193527 | bd0032ae671d873f399defe41edbb7137495b8a0af36a3cbcb5f964aa847de57

**Comment by ardabro** [comment]
If I delete this sqlite table, will it start syncing from the very first block again?

**Comment by lvcivs** [comment]
No, it will just start looking for peers from zero. But if you are connected to 52 peers then it is clearly not a connection issue, so this tip won't help you.

**Comment by litong047** [comment]
try

Connect to other peers
ip：chia.yacp.net
port：8444

**Comment by pidzwick** [comment]
I have same problem (also stuck on 193536).
From log:
17:50:34.969 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '176.36.79.195', 'port': 8444} 
17:50:43.979 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '70.181.2.63', 'port': 8444} 
17:50:54.085 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '115.159.69.123', 'port': 8444} 
17:51:01.638 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '80.66.254.171', 'port': 8444} 
17:51:07.716 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '90.226.238.64', 'port': 8444} 
18:01:16.874 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '217.224.10.151', 'port': 8444} 
18:01:25.725 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '5.129.92.153', 'port': 8444}

**Comment by lvcivs** [comment]
> I have same problem (also stuck on 193536).
> From log:
> 17:50:34.969 full_node src.full_node.full_node : ERROR Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '176.36.79.195', 'port': 8444}
> 17:50:43.979 full_node src.full_node.full_node : ERROR Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '70.181.2.63', 'port': 8444}
> 17:50:54.085 full_node src.full_node.full_node : ERROR Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '115.159.69.123', 'port': 8444}
> 17:51:01.638 full_node src.full_node.full_node : ERROR Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '80.66.254.171', 'port': 8444}
> 17:51:07.716 full_node src.full_node.full_node : ERROR Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '90.226.238.64', 'port': 8444}
> 18:01:16.874 full_node src.full_node.full_node : ERROR Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '217.224.10.151', 'port': 8444}
> 18:01:25.725 full_node src.full_node.full_node : ERROR Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '5.129.92.153', 'port': 8444}

Looks like you failed to upgrade to version 1.1.0+ in time before the freeze. You need to upgrade to the latest version to continue farming.

**Comment by lvcivs** [comment]
Which version are you on @ardabro ?

**Comment by pidzwick** [comment]
I had version 1.1.2. But I installed version 1.1.2 again (chia-blockchain_1.1.2_arm64.deb). Still same problem:
18:39:51.890 full_node src.full_node.full_node : ERROR    Error: Err.INITIAL_TRANSACTION_FREEZE, Invalid block from peer: {'host': '79.113.79.71', 'port': 8444}

**Comment by pidzwick** [comment]
I unistalled chia and installed again and it works now.

**Comment by ardabro** [comment]
I'm on 1.1.2. Nothing changed:
```
Current Blockchain Status: Full Node syncing to block 194926 
Currently synced to block: 193536
Current Blockchain Status: Not Synced. Peak height: 193536
      Time: Tue Apr 27 2021 11:42:21 CEST                  Height:     193536
```

I'll try to purge and sync from the beginning

**Comment by jgabriel98** [comment]
Same problem, on windows

reinstalling chia did not solve the issue.

the only error i get is:
```
DevTools failed to load SourceMap: Could not parse content for file:///C:/Users/jgmu6/AppData/Local/chia-blockchain/app-1.1.2/resources/app.asar/build/renderer/js/main.js.map: Unexpected end of JSON input
```

unpn is enabled, and port (8444) fowarding too.

**Comment by ardabro** [comment]
Indeed: it was about peer_table_node.sqlite; I started syncing to empty mainnet, and after some time when new peer_table_node.sqlite was populated I copied it to the old mainnet, restored it, and ran my node again. It started from block 193xxx where it stopped before, so I don't need to sync painstakingly from level zero again.

**Comment by jgabriel98** [comment]
I waited for half an hour, and it started syncing.

Don't know why

**Comment by MrClean1337** [comment]
Somehow my node wont sync anymore on peak height 194912

The gui is trying to connect to a peer. I see that below in the "full node"  window.
For about 100ms there is a new connection/peer visible and then it pops away, like instant.

So there are only 2 connections: farmer and wallet.

I just got stuck. I reinstalled chia, deleted everything, tried different versions, replaced blockchain files, deleted them and so on.
Adding a custom peer wont help...

Im using a freshly installed windows with IPv4 over IPv6 configured, because most services wont work with IPv6.
